posthog/models/session_replay_event/sql.py

```diff
@@ -117,7 +120,10 @@
 sum(console_log_count) as console_log_count,
 sum(console_warn_count) as console_warn_count,
 sum(console_error_count) as console_error_count,
-sum(size) as size
+sum(size) as size,
+-- we can count the number of kafka messages
+-- as a proxy for the number of events
+count(*) as event_count
```

Comment:
we will rename this `message_count` i.e. how much flows through kafka count `session_replay_summaries`

we will add `event_count` which is the count of rrweb_events in each session replay summary

So we can see how well we're batching, how much traffic